assets/install-scripts/install-selinux.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright 2025 Gravitational, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script installs the SELinux module for Teleport SSH.
+
+set -euo pipefail
+
+TELEPORT="teleport"
+TELEPORT_ARGS=""
+
+while getopts "c:t:" opt; do
+    case "${opt}" in
+        c)
+            TELEPORT_ARGS="-c ${OPTARG}"
+            ;;
+        t)
+            TELEPORT="${OPTARG}"
+            ;;
+        *)
+            echo "Usage: $0 [-c config_path] [-t teleport_path]"
+            exit 2
+            ;;
+    esac
+done
+
+# Write SELinux module source to a temporary directory
+WORK_DIR="$(mktemp -d)"
+"${TELEPORT}" selinux module-source > "${WORK_DIR}/teleport_ssh.te"
+"${TELEPORT}" selinux file-contexts ${TELEPORT_ARGS} > "${WORK_DIR}/teleport_ssh.fc"
```

Comment:
nit: If you're going to change the working directory to this anyway, it might be better to do that first and then run the commands

Reply:
I actually did this on purpose so that if the argument to `-t` is a local path it will work correctly